src/component/CodeBlock.tsx

```diff
@@ -13,6 +19,12 @@ darkTheme["hljs-selection"] = {
   backgroundColor: "#3a404b" // https://github.com/atom/one-dark-ui/blob/master/styles/ui-variables.less#L32
 };
 
+SyntaxHighlighter.registerLanguage("javascript", javascript);
+SyntaxHighlighter.registerLanguage("typescript", typescript);
+SyntaxHighlighter.registerLanguage("json", json);
+SyntaxHighlighter.registerLanguage("markdown", markdown);
+SyntaxHighlighter.registerLanguage("shell", shell);
```

Comment:
We could safely get rid of shell. We don't have shell scripts - seems a waste to try to highlight one-line commands.